### PR TITLE
Use Unity internal GridLayoutGroup (from UI) to position Buttons

### PR DESCRIPTION
**Pull request by brean**
Bugfix for the issue mentioned in https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/1137

### CODE REVIEW

File: Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs [hunk 1]
```diff
@@ -7,17 +7,18 @@ namespace HoloToolkit.Examples.InteractiveElements
...
         public GameObject InteractivePrefab;
 
+        [Tooltip("scale for new instance of InteractivePrefab")]
+        public Vector3 prefabScale = new Vector3(2000, 2000, 2000);
+
         [Tooltip("Data to fill the InteractiveSet.")]
         public List<string> Titles = new List<string>();
 
-        [Tooltip("number of elements after which we start a new column")]
-        public int Rows = 3;
```
Comment (keveleigh) [inline Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs]: Is there no way/need to specify a row count anymore?

This seems like a possible breaking change.

File: Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs [hunk 2]
```diff
@@ -7,17 +7,18 @@ namespace HoloToolkit.Examples.InteractiveElements
     [RequireComponent(typeof(InteractiveSet))]
     public class InteractiveGroup : MonoBehaviour
     {
+        [Tooltip("Gameobject containing GridLayoutGroup")]
+        public GameObject grid;
```
Comment (keveleigh) [inline Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs]: Please capitalize the first letter of `public` variables.

File: Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs [hunk 3]
```diff
@@ -7,17 +7,18 @@ namespace HoloToolkit.Examples.InteractiveElements
...
+
         [Tooltip("Prefab for your interactive.")]
         public GameObject InteractivePrefab;
 
+        [Tooltip("scale for new instance of InteractivePrefab")]
+        public Vector3 prefabScale = new Vector3(2000, 2000, 2000);
```
Comment (keveleigh) [inline Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs]: Please capitalize the first letter of `public` variables.

File: Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs (hunk 1 above)
Comment (brean) [reply Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs]: yes, you still can, in fact it is way better now - you can specify max rows or colums in the inspector for the grid.

Comment (keveleigh) [reply Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs]: Great! Just checked it out and it looks super handy!

**Pull request merged**